src/tail/session.rs

```diff
@@ -89,7 +89,7 @@ async fn get_tunnel_url(shutdown_rx: &mut Receiver<()>) -> Result<String, failur
     // This retry loop retries retrieving the cloudflared endpoint url from the cloudflared /metrics
     // until it gets the URL or has tried retrieving the URL over 5 times.
     impl RetryDelay {
-        fn new(max_attempts: u64) -> Self {
+        fn new(max_attempts: u64) -> RetryDelay {
```

Comment:
was always interested in `Self` vs `${StructName}` - I think I actually like `Self` better so you don't have to update a billion instances but won't block on